package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.99.99",
   "description": "The premier osquery fleet manager.",
   "engines": {
-    "node": "20.18.1",
```

Comment:
Was this intentionally committed?  While I definitely think it's great to update this, I just wanted to limit the scope of changes in the PR to what the ticket states if possible. Or if the change is the PR require it, just a small comment about why would be great.